models/classes/StatisticalMetadata/Import/Extractor/MetadataAliasesExtractor.php

```diff
@@ -29,18 +29,40 @@ class MetadataAliasesExtractor
     /** @var MetadataHeadersExtractor */
     private $metadataHeadersExtractor;
 
+    /** @var array */
+    private $cache = [];
+
     public function __construct(MetadataHeadersExtractor $metadataHeadersExtractor)
     {
         $this->metadataHeadersExtractor = $metadataHeadersExtractor;
     }
 
     public function extract(array $header): array
     {
-        return array_map(
-            static function (string $metadataHeader): string {
-                return str_replace(Header::METADATA_PREFIX, '', $metadataHeader);
-            },
-            $this->metadataHeadersExtractor->extract($header)
-        );
+        $hash = $this->getHash($header);
+
+        if (!array_key_exists($hash, $this->cache)) {
```

Comment:
`extractAliases` would at least return an empty array, so if a key exists it would at least be set and be an empty array. Therefore, I think we can use `isset` instead of `array_key_exists` here, as it returns true if the key exists even if its value is considered empty

```suggestion
        if (!isset($this->cache[$hash])) {
```

(Just checked :D)
```
~$ php
<?php $a=['k'=>[]]; var_dump(isset($a['k']));
Standard input code:1:
bool(true)
```